fabtests/benchmarks/benchmark_shared.c

```diff
@@ -283,6 +283,13 @@ int run_pingpong(void)
 			return ret;
 	}
 
+	/* ft_finalize() needs RX buffer to be posted for proper sync */
```

Comment:
Should this be moved to `pingpong_no_pre_posted_rx` ?

Reply:
@iziemba can you take a look at this change ?

Reply:
Not that easy to move it to `pingpong_no_pre_posted_rx()`. The function is called from `pingpong()`. Pingpong is called from two places above. One of them in a loop. We need that at the end of the last iteration only.

Reply:
Ah, u are right.